Cargo.lock

```diff
@@ -1472,9 +1472,9 @@ checksum = "37b2a672a2cb129a2e41c10b1224bb368f9f37a2b16b612598138befd7b37eb5"
 
 [[package]]
 name = "cc"
-version = "1.2.12"
```

Comment:
Do we need to do `cargo update`? Maybe we can explode it from this pr?

Reply:
The goal was to update derive more to the stable "1" version instead of "0.xx" but will remove from this PR then